react/week-2/lesson.md

```diff
@@ -520,15 +535,23 @@ This is a very common pattern which will come in very useful!
 | 12. Still within the second `.then` callback function, update the `pokemon` state variable. <details><summary>Click here is you are stuck.</summary>:ppl at the State section again to see how to set state variables to new values.</details> |
 | 13. What happens in your browser? Do you understand why? If not, discuss it with another student. If you are both stuck, ask a Teaching Assistant. |
 
+### Container components
+
+In real world applications, the things we want to remember in state follow the *business logic* required by our users. So for example the number of caught Pokemon in the exercise  increases when you click on the button *Catch Pokemon*. Most of the time, business logic is about figuring out when and how to change state.
```

Comment:
Will the students understand the term 'business logic' or should we be more explicit here about what that is?

Reply:
Yes good point. Probably worth a link to a definition.

Reply:
Done in https://github.com/CodeYourFuture/syllabus/pull/448/commits/a247ddff9371df276917aa36d9760df26fa6c82f